defopt.py

```diff
@@ -25,10 +25,10 @@
 from sphinxcontrib.napoleon.docstring import GoogleDocstring, NumpyDocstring
 import typing_inspect as ti
 
-try:
+try:  # pragma: no cover
```

Comment:
necessary to keep coverage up for Py2 (excluding the same lines for Py2 and Py3).